index.html

```diff
@@ -91,6 +93,41 @@ <h2> <a href="supermarkets.html" class="badge">Search now</a></h2>
                 </div> 
         </div>
     </div>
+
+    <!---Modal section-->
+    <div class="modal fade" id="cart" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
```

Comment:
`aria-labelledby="exampleModalLabel"` - I don't think you need this. Because you're not going to make this site for screen readers. I think it's just copy-paste leftovers